docs/user/man_pages/bodhi-push.rst

```diff
@@ -65,4 +69,4 @@ request::
 
     https://github.com/fedora-infra/bodhi
 
-Bodhi's documentation is available online: https://bodhi.fedoraproject.org/docs
\ No newline at end of file
+Bodhi's documentation is available online: https://bodhi.fedoraproject.org/docs
```

Comment:
I'm going to force push to your branch to remove this change.

Reply:
It looks like vim is adding new line by default(hedump -> 0x0a) I see that all files in man_pages have 0a on the end. Do you think we should we left this file without newline at the end of file?